src/providers/bee.tsx

```diff
@@ -1,25 +1,26 @@
-import { createContext, ReactChild, ReactElement, useEffect, useState } from 'react'
+import { createContext, ReactChild, ReactElement } from 'react'
 import { Bee, Data, FileData, Reference } from '@ethersphere/bee-js'
-import { UPLOAD_HOSTS, META_FILE_NAME, POSTAGE_STAMP, DOWNLOAD_HOST, PREVIEW_FILE_NAME } from '../constants'
+import { BEE_HOSTS, META_FILE_NAME, POSTAGE_STAMP, PREVIEW_FILE_NAME } from '../constants'
 
-const randomHost = UPLOAD_HOSTS[Math.floor(Math.random() * UPLOAD_HOSTS.length)]
-const bee = new Bee(randomHost)
-const beeGateway = new Bee(DOWNLOAD_HOST)
+const randomIndex = Math.floor(Math.random() * BEE_HOSTS.length)
+const randomBee = new Bee(BEE_HOSTS[randomIndex])
 
 interface ContextInterface {
   isConnected: boolean
   upload: (file: File, preview?: Blob) => Promise<Reference>
   getMetadata: (hash: Reference | string) => Promise<Metadata | undefined>
   getPreview: (hash: Reference | string) => Promise<FileData<Data>>
   getChunk: (hash: Reference | string) => Promise<Data>
+  getDownloadLink: (hash: Reference | string) => string
 }
 
 const initialValues: ContextInterface = {
-  isConnected: false,
+  isConnected: true,
```

Comment:
we should remove this (it actually is in the other PR)

Reply:
Removed it